internal/services/containers/kubernetes_cluster_resource.go

```diff
@@ -642,6 +642,12 @@ func resourceKubernetesCluster() *pluginsdk.Resource {
 				Default:  false,
 			},
 
+			"run_command_enabled": {
+				Type:     pluginsdk.TypeBool,
+				Optional: true,
+				Default:  true,
```

Comment:
since this is a security setting, should this be set to `false` by default?

Reply:
I share the same idea, and I expressed it in another MS issue. This is however the default for all AKS clusters, it is an opt out feature. Which is why I want to add this parameter so I can shut if off for all of my clusters. My personal opinion is that going against Azure defaults in the provider for security reasons may be nice but not smart. However correct me if I am wrong if this is standard practice :smile:

Reply:
@phillebaba we discussed this briefly offline, we can leave this as is.